Sources/MongoSwift/Operations/MongocCommandHelpers.swift

```diff
@@ -0,0 +1,58 @@
+import CLibMongoC
+
+/// Executes the provided closure using a stack-allocated, mutable bson_t. The bson_t is only valid for the body of the
+/// closure and must be copied if you wish to use it later on.
```

Comment:
IIUC It may be worth noting that the closure must initialize the bson_t, otherwise the deferred `bson_destroy` would be accessing uninitialized memory (unless `bson_t()` also initializes).

Reply:
So according to [this](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/using_imported_c_structs_and_unions_in_swift):
> If all imported members have default values, Swift also provides a default initializer that takes no arguments.

that's what we're using here. using the debugger it appears this just creates a totally zeroed-out `bson_t`. so yes, I believe we do have to initialize the `bson_t` in the closure.